Sources/CNIOAtomics/include/CNIOFastAtomics.h

```diff
@@ -0,0 +1,280 @@
+//===----------------------------------------------------------------------===//
```

Comment:
The addition of this file is going to break the `swift package generate-xcodeproj` ability to generate Xcode modules (I know it's weird, just trust me on this point). Can we add these definitions to `CNIOAtomics.h` instead?

Reply:
Sure, moved.